chameleonultragui/lib/l10n/app_en.arb

```diff
@@ -167,6 +167,8 @@
   "dump_partial_data": "Dump partial data",
   "additional_key_dict": "Additional key dictionary",
   "check_keys_dict": "Check keys from dictionary",
+  "check_keys_autopwn": "Try automatic recovery",	
```

Comment:
dont just call the key smthng with autopwn, when translating noone will know the context... Also; very long button name...

Reply:
i mean the button above is "Check keys from dictionary". thats a bit longer then "Try automatic recovery".

but idc, change it how you want